client/src/components/exercise/lab0/DesignLabSection/ScrumActivity/ScrumBoardActivity.js

```diff
@@ -20,28 +20,25 @@ const ScrumBoardActivity = () => {
       <h2 className={"tw-title tw-text-left"}> Sprint Planning Activity</h2>
       <p className={"tw-py-3 tw-body-text"}>
         In this exercise you will be given the opportunity to plan your
-        development schedule of the lab using the knowledge you’ve learned about
-        SCRUM and Agile development.
+        development schedule of the Accessibility to Focus Order exercise that
+        you just created in the previous activity.
       </p>
       <p className={"tw-body-text"}>
-        Using the blocks below, drag and drop the blocks in the correct
-        sequence.
+        Using the blocks below, drag and drop them into the correct sprint
+        sequence based on the stories that are present in each block.
```

Comment:
Maybe add some more context about what needs to be developed and in what order. I had to think about what order the sprints should be in. Or maybe just have a more detailed description of what the focus order exercise is.

![Screenshot 2025-03-17 at 5 16 02 PM](https://github.com/user-attachments/assets/1e62a08a-ef67-431c-b862-534efda017b0)